modules/candidate_parameters/README.md

```diff
@@ -0,0 +1,51 @@
+# Candidate Parameters
+
+## Purpose
+
+The Candidate Parameters module allows users to input 
+candidate data that is not timepoint specific.
+
+## Intended Users
+
+The primary types of users are:
+1. Data entry personnel
+2. Site coordinators validating the data
+
+## Scope
+
+The Candidate Parameters module stores all information 
+for a candidate that are not timepoint dependent. All data 
+being collected during a visit should be entered as an instrument 
+at the visit level.
+
+## Permissions
+
+The Candidate Parameters module uses the following permissions. Any one of them
+is sufficient to have access to the module.
+
+ - candidate_parameter_view
+    - This permission gives the user viewing access to the module and all its tabs.
+
+ - candidate_parameter_edit
+    - This permission gives the user editing access to the module and all its tabs.
+
+## Configurations
+
+The candidate parameters module has the following configurations that affect its usage
```

Comment:
Missing mention of the `participant_status_options` table behaviour, and the config.xml `<consent>` tag (and subtags) that goes along with `useConsent`.

The wiki page that christine mentioned also says that there's a `<parameter_candidate>` tag, but I don't know if that works. (Same with the "Candidate Parameters" magic parameter_category.. I don't know if that still works, but if it does and affects what shows up on the Candidate Info subpage it should be documented here..)

Reply:
done